ScoutSuite/providers/gcp/configs/services.py

```diff
@@ -31,7 +36,7 @@ def __init__(self, metadata=None, thread_config=4, projects=None, **kwargs):
         except NameError as e:
             pass
 
-        self.stackdriverlogging = StackdriverLoggingConfig(thread_config=thread_config)
+        self.stackdriverlogging = StackdriverLogging(gcp_facade, stackdriverlogging_facade)
         # self.stackdrivermonitoring = StackdriverMonitoringConfig(thread_config=thread_config)
```

Comment:
Why is this commented?

Reply:
It was this way before, I imagine it is because the Stackdriver monitoring config does nothing on the public repo

Reply:
Removed it!